cd/auto-generate/auto-generate-controllers.sh

```diff
@@ -230,7 +230,7 @@ for CONTROLLER_NAME in $CONTROLLER_NAMES; do
     # Add all the files & create a GitHub commit
     git add .
     COMMIT_MSG="Update ACK runtime to \`$ACK_RUNTIME_VERSION\`
-    release notes: $CODEGEN_RELEASES/tag/$ACK_RUNTIME_VERSION"
+    Release notes: $CODEGEN_RELEASES/tag/$ACK_RUNTIME_VERSION"
```

Comment:
Please still add an empty line - git uses this to differentiate between the status message and the full description.

Reply:
This breaks up the commit as follows:
![image](https://user-images.githubusercontent.com/61433408/148802525-0f362135-e044-4a07-9431-48ceff80b491.png)

Are you saying it should render differently?